less/index.less

```diff
@@ -1,59 +1,244 @@
-// Is this working?
-/* http://meyerweb.com/eric/tools/css/reset/ 
-   v2.0 | 20110126
-   License: none (public domain)
-*/
-
-html, body, div, span, applet, object, iframe,
-h1, h2, h3, h4, h5, h6, p, blockquote, pre,
-a, abbr, acronym, address, big, cite, code,
-del, dfn, em, img, ins, kbd, q, s, samp,
-small, strike, strong, sub, sup, tt, var,
-b, u, i, center,
-dl, dt, dd, ol, ul, li,
-fieldset, form, label, legend,
-table, caption, tbody, tfoot, thead, tr, th, td,
-article, aside, canvas, details, embed, 
-figure, figcaption, footer, header, hgroup, 
-menu, nav, output, ruby, section, summary,
-time, mark, audio, video {
-	margin: 0;
-	padding: 0;
-	border: 0;
-	font-size: 100%;
-	font: inherit;
-	vertical-align: baseline;
-}
-/* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
-	display: block;
-}
-body {
-	line-height: 1;
-}
-ol, ul {
-	list-style: none;
-}
-blockquote, q {
-	quotes: none;
-}
-blockquote:before, blockquote:after,
-q:before, q:after {
-	content: '';
-	content: none;
-}
-table {
-	border-collapse: collapse;
-	border-spacing: 0;
-}
+@import 'reset.less';
+@import 'variables.less';
+@import 'mixins.less';
 
-/* Set every element's box-sizing to border-box */
-* {
-    box-sizing: border-box;
+:root {
+	font-size: 62%;
+	background-image: url('../images/bg.png');
 }
 
-html, body {
-    height: 100%;
-}
+.main-container {
+	.flex(column, 0, 0);
```

Comment:
This is fine, but I feel all the section below is nested too deeply and too long. It becomes not really easy to read, especially if you need to check something at the bottom of the section (and you need to scroll all the way up).
You could want to refactor it with cleaner code.